modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -1,34 +1,36 @@
+NOTE: User key-based encryption is planned to be removed from ownCloud in the near future. While it is a bit more secure than a central encryption approach, User key-based encryption has some disadvantages. It blocks some additional functions such as the integration of an online editor like LibreOffice or OnlyOffice into ownCloud and can cause problems when sharing files with groups. Therefore Master-key-based encryption is now the recommended setup fr all new insallations.
 
-If you need to migrate from User Key-based to Master Key-based encryption, there are several steps that you need to follow to ensure a smooth and complete transition.
-These are:
+The decryption workflow described here works for both MySQL and SQLite based databases. It will only work when users have enabled the password recovery. Moreover, an admin recovery password needs to be activated and available for the ownCloud administrator.
+
+If you need to migrate from User Key-based to Master Key-based encryption, there are several steps that you need to follow to ensure a smooth and complete transition:
 
 . xref:disable-user-key-based-encryption[Disable User Key-based encryption]
 . xref:remove-the-encryption-records-from-the-owncloud-database[Remove the encryption records from the ownCloud database]
 . xref:remove-the-files_encryption-directory[Remove the `files_encryption` directory]
 . xref:encrypt-the-filesystem-using-master-key-based-encryption[Encrypt the filesystem using Master Key-based encryption]
 
-include::{partialsdir}/configuration/server/enable-single-user-mode.adoc[leveloffset=+1]
 
 [[disable-user-key-based-encryption]]
 == Disable User Key-based Encryption
 
-The first part of the migration process is to take is to decrypt all files and to disable encryption in ownCloud, which requires three commands to be executed.
+The first part of the migration process is to decrypt all files and to disable encryption in ownCloud, which requires three commands to be executed.
 These commands are: xref:configuration/server/occ_command.adoc#encryption[`occ encryption:decrypt-all` and `occ encryption:disable`], and xref::configuration/server/occ_command.adoc#apps-commands[`occ app:disable`].
 
 You can see an example of calling the commands listed below, configured to require no user interaction.
 
 [source,console]
 ----
-sudo -u www-data occ encryption:decrypt-all --continue=yes && \
-  sudo -u www-data occ encryption:disable --no-interaction && \
-  sudo -u www-data occ app:disable --no-interaction encryption
+sudo -u www-data php occ encryption:decrypt-all --continue=yes && \
+  sudo -u www-data php occ encryption:disable --no-interaction && \
+  sudo -u www-data php occ app:disable --no-interaction encryption
 ----
 
+NOTE: The decryption of the files by the central administration requires the current passwords of all users! 
```

Comment:
Where you've written "central administration", did you mean the systems administrator or ownCloud administrator?

Reply:
the owncloud administrator

Reply:
I would add: only work when users have enabled password recovery and if an admin recovery password is available